tensorflow_io/core/python/experimental/avro/python/utils/__init__.py

```diff
@@ -0,0 +1 @@
+# This is a package
```

Comment:
Let's use an empty file for this. You can delete the `__init__.py`, then use `touch __init__.py` to create an empty file.

Reply:
updated